templates/example_cell_clustering.ipynb

```diff
@@ -17,7 +17,9 @@
   {
```

Comment:
<p>Same here, cell_cluster_col is no longer needed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/460/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>